portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro

```diff
@@ -5054,6 +5054,34 @@ definition {
 		Portlet.viewNoPermissionPG();
 	}
 
+	macro viewRemoteApp {
+		task ("Assert Remote App table entry") {
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.viewTableEntries(
+				entryName = "${upgradeName}",
+				entryType = "${upgradeType}");
+		}
+
+		task ("Assert Remote App Name, Type, and URL fields") {
+			Click(
```

Comment:
Try and use LexiconEntry.gotoEntry

Reply:
hmmm LexiconEntry.gotoEntry and LexiconEntry.gotoEntryMenuItem dont work because the remote app rows dont start with **//*[@data-qa-id='row']**//*[normalize-space(text())='Test Remote App'], seems like the existing tests in RemoteApps.testcase also just use Click()

Reply:
Thanks for trying it out, ok to use Click for now. Would like to refactor to use a more generic macro.